README.md

```diff
@@ -417,6 +417,18 @@ This method was introduced in PHPUnit 10.0.0.
 
 [`Assert::assertIsList()`]: https://docs.phpunit.de/en/main/assertions.html#assertislist
 
+#### PHPUnit < 10.1.0: `Yoast\PHPUnitPolyfills\Polyfills\AssertIsList`
```

Comment:
Wrong trait name

Reply:
Ow! Thanks for catching that @garak !

Reply:
(and yes, I have fixed it locally and that commit will be included in the 2.0.0 release.)